README.md

```diff
@@ -8,60 +8,64 @@ Intended for [gettext](https://www.npmjs.com/search?q=gettext) or similar transl
 
 - Two functions
   - `gettext`
-    - Simple translation with language strng as lookupkey
+    - Simple translation with template string as translation lookup key
   - `ngettext`
-    - Singular/Plural aware translation with language strng as lookupkey
+    - Singular/Plural aware translation with template string as translation lookup key
 - Two ways of invoking
-  - simple values
-    - The argument/string is used as the lookup id
-  - object
-    - The key is used as the lookup id
+  - direct substitutions (simple values)
+    - Immediately substituted, same as normal template literals.
+    - Occurs **before** translation.
+    - Only permitted for primitives `string`, `number`, `boolean`, etc.
+  - object substitutions (single `key`/`value`)
+    - The `key` gives your translator context (and needs to survive translation).
+    - The `value` is substituted after **translation**.
 ### `gettext`
 
-Simple Values:
+- Without Substitution:
 
 ```js
 gettext`foo`
 ```
 
-key/val:
+- With Substitution:
 
 ```js
-gettext`Welcome to ${{location: 'hell'}}`
+gettext`Welcome to ${{location: 'Jurassic Park'}}`
 ```
 
-jsx (nested call):
+- jsx:
 
 ```js
 gettext`For more ${{link: <a href="http://google.com">{gettext`information`}</a>}}`
 ```
 
 ### `ngettext`
 
-- simple value:
+- Without Substitution:
 
 ```js
 ngettext(planets)`Hello world!|Hello worlds!`
 ```
 |   | input | output |
 |---|----|----|
-| **singular** | planets <= 1  | "Hello World!"  |
-| **plural**   | otherwise     | "Hello Worlds!" |
+| **singular** | `planets === 1`  | "Hello World!"  |
+| **plural**   | `planets > 1`    | "Hello Worlds!" |
 <!-- Basic logic: If `planets > 1`  return "Hello world" otherwise -->
 
-- Object:
+- With Substitution:
 
 ```js
 ngettext(daysLeft)`Only one day to go!|${{daysLeft}} days to go!`
 ```
 
 |   | input | output |
 |---|----|----|
-| **singular** | `{daysLeft: 1}`  | "Only one day to go!"  |
-| **plural**   | `{daysLeft: 9}`  | "9 days to go!" |
+| **singular** | `daysLeft === 1`  | "Only one day to go!"  |
+| **plural**   | `daysLeft > 1`    | "9 days to go!"        |
```

Comment:
I think this needs to be `daysLeft === 9` given what you have on the righe

Reply:
Ah yeah - good point....hmmm not quite sure quite sure which bit to change actually 🤔

Reply:
I think you were right the first time. You just need to communicate what value `daysLeft` is being assigned. So maybe single equals or the triple equals. But either way line 1 is `1` and line 2 is `9`.

Reply:
lol after running this it doesnt seem to behave like i had described either...

Reply:
Well just let me know if you see any problems with its properties, that could lead to other PRs 🤔